flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1232,21 +1232,31 @@ def run_to_convergence(self, desired_datasets, max_iterations=4):
 
     @given(
```

Comment:
Decided to do this right. I'll probably send back out for review if I can get a quick glance over it.